src/components/Logo/Logo.js

```diff
@@ -8,14 +8,120 @@ export default function Logo() {
       {/* <i className="fa d-inline fa-lg fa-stop-circle-o" /> */}
       <div className="float-left ml-1 mr-0" style={{ width: 35 }}>
         <svg
```

Comment:
why not saving file as svg?

Reply:
From what I remember whenever you take assets and you insert them into your page it's always a bit tricky if what you are trying to add is not a .js file. Therefore it's all in there... There might be better ways, I'm sure ...